.changeset/tame-candles-wash.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-connectors/fuel-development-wallet": minor
+"@fuel-connectors/fuelet-wallet": minor
+"@fuel-connectors/fuel-wallet": minor
+"@fuels/connectors": minor
+---
+
+feat: bump to @fuels/react 0.17.0 which moved react-query to peerDependency.
```

Comment:
I think we don't need this changeset.. as the @fuels/react is only used by the example app.

Reply:
https://github.com/FuelLabs/fuel-connectors/pull/11/commits/107462207c19941622ee97f0dd21c2d5834b16b5